portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,24 +35,37 @@
     
     public class Name 
     {
-        private ArrayList<String> names = new ArrayList<String>();
+        public ArrayList<String> names = new ArrayList<String>();
     }
   
-    private Name name = new Name();
+    Name name = new Name();
+   
 
-  
+    
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Name name = new Name();
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         Gson gson = new Gson(); 
+        Query query = new Query("Names");
+        PreparedQuery results = datastore.prepare(query);
+        
+        for (Entity entity : results.asIterable()) 
+        {
+            name.names.add( (String) entity.getProperty("viewer0"));
+        }
```

Comment:
I still don't fully understand this iteration. Can we go over this next meeting?